src/mh-core.jl

```diff
@@ -46,7 +46,9 @@ struct MetropolisHastings{D} <: MHSampler
 end
 
 StaticMH(d) = MetropolisHastings(StaticProposal(d))
+StaticMH(d::Int) = MetropolisHastings(StaticProposal(MvNormal(zeros(d), ones(d))))
```

Comment:
```suggestion
StaticMH(d::Int) = MetropolisHastings(StaticProposal(MvNormal(FillArrays.Zeros(d), I)))
```
where `I` is from `LinearAlgebra` and I'm slightly uncertain if `FillArrays` is already loaded, so you might have to do that.

Reply:
These are lazy representations, and so the resulting code will have better performance and memory characteristics.

Reply:
I love this suggestion